www/form/inner.js

```diff
@@ -5099,9 +5136,30 @@ define([
                 editableStr
             ]);
 
+            var toggleOffclass = 'ontouchstart' in window ? 'toggle-active' : 'toggle-inactive'; 
+            var toggleOnclass = 'ontouchstart' in window ? 'toggle-inactive' : 'toggle-active'; 
+            var toggleDragOff = h(`button#toggle-drag-off.cp-form-view-drag.${toggleOffclass}.fa.fa-arrows`, {'aria-hidden': true, 'title': Messages.toggleArrows});
+            var toggleDragOn = h(`button#toggle-drag-on.cp-form-view-drag.${toggleOnclass}.fa.fa-hand-o-up`, {'aria-hidden': true, 'title': Messages.toggleDrag});
+            $(toggleDragOff).click(function() {
+                $(toggleDragOff).attr('class').indexOf('toggle-inactive') !== -1 ? $(toggleDragOff).toggleClass('toggle-inactive').toggleClass('toggle-active') && $(toggleDragOn).toggleClass('toggle-active').toggleClass('toggle-inactive') : undefined;
+                updateForm(framework, content, true);
+                APP.mainSortable.options.disabled =  true;
```

Comment:
Here, if you keep redrawing everytime with "updateForm", you probably don't need to update `APP.mainSortable` (it would be recreated anyway). If you update the code to avoid redrawing the entire form everytime, you need to replace the `updateForm` line above.